Makefile

```diff
@@ -69,6 +69,7 @@ install-sylius:
 	${CONSOLE} sylius:fixtures:load default -n
 	${YARN} install
 	${YARN} build
+	${CONSOLE} assets:install -n
```

Comment:
i'm wondering if `composer req` does not already play this command at [auto-scripts](https://github.com/Sylius/Sylius-Standard/blob/1.14/composer.json#L126) :thinking: